crates/euclid_wasm/src/lib.rs

```
@@ -472,3 +477,41 @@ pub fn get_payout_description_category() -> JsResult {
 
     Ok(serde_wasm_bindgen::to_value(&category)?)
 }
+
+#[wasm_bindgen(js_name = getValidWebhookStatus)]
+pub fn get_valid_webhook_status(key: &str) -> JsResult {
+    let event_class =
+        EventClass::from_str(key).map_err(|_| "Invalid webhook event type received".to_string())?;
```

Don't discard the error, log the error

---

We are not loging any errors from WASM binaries, so kept it this way. Calling this function with an incorrect variant will throw an error - will need to be caught in `trycatch` block during integration